pyGPG/gpg.py

```diff
@@ -159,6 +220,18 @@ def decrypt(self, inputtxt=None, inputfile=None, outputfile=None):
         '''
         return self.runGPG('decrypt', inputtxt, inputfile, outputfile)
 
+<<<<<<< HEAD
+=======
+    def encrypt(self, inputtxt, recipient):
+        '''encrypts the inputtxt block passed in
+        and/or returns the decrypted as GPGResult.output
+
+        @param inputtxt: string (optional)  of text to send to gpg's stdin
+        @rtype GnuPGResult object
+        '''
+        return self.runGPG2('encrypt', inputtxt, '--recipient',recipient, '--trust-mode', 'always', '--armor')
+
```

Comment:
ok, now it makes sense why you made runGPG2.

But I think you are missing the basic idea I had when I created this lib.  All the other python libraries that existed only covered a few gpg capabilities and were fixed in the options used.  I created this lib to be config driven.  In that way, this base library could be small lightweight, but also capable of running nearly any gpg command just by modifying the options for a particular task.

This function can be modified to use that mehtod and then use the original runGPG().
That would be a reason to add this convenience function.  As it would be common to most any user application using this lib.